src/routes/Overview.js

```diff
@@ -64,31 +61,18 @@ const Overview = ({ loadApis, services, history, selectRow, onError }) => {
                                                 />
                                             </LevelItem>
                                             <LevelItem>
-                                                <Dropdown
-                                                    dropdownItems={ [
-                                                        <DropdownItem
-                                                            key="download"
-                                                            component="button"
-                                                            onClick={ () => {
-                                                                multiDownload(services.selectedRows, onError);
-                                                            } }
-                                                        >
-                                                            Download selected
-                                                        </DropdownItem>
-                                                    ] }
-                                                    isOpen={ isOpen }
-                                                    onSelect={ () => onOpenToggle(!isOpen) }
-                                                    toggle={
-                                                        <KebabToggle
-                                                            onToggle={ (isOpen) => onOpenToggle(isOpen) }
-                                                            isDisabled={ !services.selectedRows ||
-                                                                Object.values(services.selectedRows || {})
-                                                                .map(({ isSelected }) => isSelected)
-                                                                .filter(Boolean).length === 0 }
-                                                        />
-                                                    }
-                                                    isPlain
-                                                />
+                                                <Button
+                                                    isDisabled={ !services.selectedRows ||
+                                                        Object.values(services.selectedRows || {})
+                                                        .map(({ isSelected }) => isSelected)
+                                                        .filter(Boolean).length === 0 }
+                                                    onClick={ () => {
+                                                        multiDownload(services.selectedRows, onError);
+                                                    } }
+                                                    variant="plain"
```

Comment:
I would actually use default `primary` variant here because plain one with text looks like it's constantly disabled.